iceoryx2/src/lib.rs

```diff
@@ -370,6 +433,30 @@
 //! # }
 //! ```
 //!
+//! ## Blackboard
+//!
+//! For a detailed documentation see the
+//! [`blackboard::Creator`](crate::service::builder::blackboard::Creator)
+//!
+//! ```
+//! use iceoryx2::prelude::*;
+//!
+//! # fn main() -> Result<(), Box<dyn core::error::Error>> {
+//! let node = NodeBuilder::new().create::<ipc::Service>()?;
+//!
+//! let service = node.service_builder(&"My/Funk/ServiceName".try_into()?)
```

Comment:
It is not so intuitively clear that `u64` is here the key type. Could you please add a comment here and mention this.

Or even better, create a type alias `type KeyType = u64` and use it here, then you do not need a comment. Could you adjust also the other examples accordingly.